lib/services/elasticsearch.js

```diff
@@ -38,6 +38,60 @@ function ElasticSearch (kuzzle, options, config) {
     }
   });
 
+  this.errorMessagesMapping = [
+    {
+      // [illegal_argument_exception] object mapping [titi] can't be changed from nested to non-nested
+      regex: /^\[illegal_argument_exception\] object mapping \[(.*?)\] can't be changed from nested to non-nested$/,
+      replacement: 'Can not change mapping for field "$1" from nested to another type'
+    },
+    {
+      // [illegal_argument_exception] object mapping [baz] can't be changed from non-nested to nested
+      regex: /^\[illegal_argument_exception\] object mapping \[(.*?)\] can\'t be changed from non-nested to nested$/,
+      replacement: 'Can not change mapping for field "$1" from object to another type'
```

Comment:
OK, i did not notice that "nested" was considered as a specific type by ES.
But as I understand the original ES error message, "from non-nested to nested", it means that this error can occur when we try to convert an `object` to a `nested`, but also when we try to convert a scalar type to a `nested`
So "from object to another type" is not exactly what it means... better say "from object or a scalar type to nested", or something like this?